docgen/src/guide/Multi_index.md

```diff
@@ -6,22 +6,19 @@ category: guide
 navWeight: 50
 ---
 
-You can use multiple [`<InstantSearch>`](guide/<InstantSearch>.html) instances for cases like:
+Whenever you want to:
 
-* displaying hits from different indices
-* sharing a single SearchBox
-* any use case involving synchronizing widgets between different [`<InstantSearch>`](guide/<InstantSearch>.html) instances
+* display hits from different indices
+* share a single SearchBox
```

Comment:
Instead of this: "build an autocomplete menu targeting different indices " I would go for "building an autocomplete using an external component". Right now I'm building the demo/recipe to build one without the need of this special connector.